.ado/publish.yml

```diff
@@ -5,38 +5,156 @@
 variables:
   Ver.Major: 0
   Ver.Minor: 0
-  Ver.Patch: 0
+  Ver.Patch: 4
 
 name: $(BuildId)
 
 trigger: none
 pr: none
 
 jobs:
-- job: "Publish"
+- job: "Build"
+  strategy:
+    matrix:
+      #linux:                           # 2023.06.30 ESRP Publishing of Py packs is not supported for linux_x86_64.
+      #  imageName: 'ubuntu-latest'     # TODO(kuzminrobin): Manilinux.
+      mac:
+        imageName: 'macOS-latest'       # MacOS-specific Py + VSCode Ext
+      windows:
+        imageName: 'windows-latest'     # Win-specific Py + Plkatform-independent Py
   pool: 
-    vmImage: 'ubuntu-latest'
+    vmImage: $(imageName)
+
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '3.11'
 
   - script: |
       python ./prereqs.py --install
+    displayName: Install Prereqs
+
+  - script: |
       npm install -g @vscode/vsce
-    displayName: Install VSCE Pub Prereqs
-    
+    displayName: Install Prereqs for VSCode Extension
+    condition: eq(variables['Agent.OS'], 'Darwin')
```

Comment:
The Mac pool is usually quite limited. Can this run on Linux or Windows instead?

Reply:
Moving to Win.
FYI: After the move the Mac will become available 10 minutes earlier, Win will take 10 minutes longer, and overall execution too. Overall execution time will increase from 15 to 25 minutes.